### PR TITLE
Update outputs in guacamole porter/terraform

### PR DESCRIPTION
**Pull request by tamirkamara**
# PR for issue #663 

## What is being addressed

Syncing both terraform and porter outputs to allow them to be fetched by the resource processor (separate task)

## How is this addressed

- Rename outputs to be exactly the same in porter and terraform
- Have outputs defined in porter with the right action (install)
- Upgrade TF to 1.0.4 
- Remove ACR from core RG

### CODE REVIEW

**Review by marrobi — APPROVED** [review]
LGTM

Are a couple of things we should think about - such as output naming we need a convention - for example always use `connection_uri` as when building the UI need to be consistent across resources.

File: workspaces/services/guacamole-azure-win10vm/porter.yaml [hunk 1]
```diff
@@ -69,6 +70,7 @@ install:
         container_name: "{{ bundle.parameters.tfstate_container_name }}"
         key: "{{ bundle.parameters.tre_id }}-ws-{{ bundle.parameters.workspace_id }}-svc-guacamole-winwin10vm"
       outputs:
+      - name: guacamole_win10vm_address
```
Comment (tamirkamara) [inline workspaces/services/guacamole-azure-win10vm/porter.yaml:73]: Here too?

File: workspaces/services/guacamole/porter.yaml [hunk 2]
```diff
@@ -41,32 +40,34 @@ parameters:
     default: false
 
 outputs:
-  - name: guacamole_workspace_name
+  - name: guacamole_url
```
Comment (tamirkamara) [inline workspaces/services/guacamole/porter.yaml:43]: connection_uri?

File: workspaces/services/guacamole-azure-win10vm/porter.yaml (hunk 1 above)
Comment (marrobi) [reply workspaces/services/guacamole-azure-win10vm/porter.yaml:73]: What's this used for?

**Review by marrobi — APPROVED** [review]
LGTM

**Pull request merged**